kedro_snowflake/starters/snowflights/prompts.yml

```diff
@@ -57,4 +57,16 @@ snowflake_password_env_variable:
     Alternatively, you can re-configure the plugin later to use Kedro's credentials.yml
   regex_validator: "^[A-Za-z0-9\\_]+$"
   error_message: |
-    It must contain only alphanumeric symbols and underscores
\ No newline at end of file
+    It must contain only alphanumeric symbols and underscores
+
+pipeline_name:
+  title: "Pipeline name used as a Snowflake task prefix"
+  regex_validator: "^[\\w-]+$"
+  error_message: |
+    It must contain only alphanumeric symbols and hyphens
+
+enable_mlflow_integration:
+  title: "Enable MLflow integration (see documentation for the configuration instructions)"
+  regex_validator: "^(True|False)"
```

Comment:
It's weird to have it typed with uppercase string. Will it work if we do `true|false` downstream?

Reply:
doesn't work, but we can change them to lower case or work with both